src/edu/psu/sweng500/team8/gui/CellGUI.java

```diff
@@ -119,7 +119,49 @@ public void populate(Cell cell, GameSession gameSession, boolean isRefresh,
 				this.numberInputField.setFocusable(true);
 
 			} else {
-				cellLostFocus(null);
+				/* Pencil Marks */
+				this.numberInputCell.setVisible(false);
+				this.pencilMarkDisplayCell.setVisible(true);
+				
+				for (int i = 1; i <= 9; i++) {
+					
+					int rowIndex = ((i - 1) / 3);
+					int columnIndex = ((i - 1) % 3);
+
+					this.pencilMarkDisplay[rowIndex][columnIndex]
+							.setVisible(true);
+					this.pencilMarkDisplay[rowIndex][columnIndex]

```

Comment:
1) What do we need the mouse adapter for again? The pencil marks are no longer clickable...
2) I'm fairly certain his will subscribe the same mouse adapter multiple times to the same control. Thus each mouse event will call the mouse adapter N times rather than once.